### ISSUE TITLE
Plugin is still in Pending Approval

### ISSUE THREAD

**Issue by Simply007** [gradle/plugin-portal-requests#52]
I have published a plugin a couple of days ago, but I didn't get any response.
If there is any mistake in the plugin, I hope to get feedback via email, but I didn't get any.

### Expected Behavior

Any feedback sent via email.

### Current Behavior

The plugin is still in "Pending approval" state

### Context

My idea was to publish plugin under my personal space name (`com.github.simply007.kontent.delivery.generator) for testing purposes and once everything is polished I wanted to republish the plugin under the new name `com.github.kentico.kontent.delivery.generator` and deprecate the testing one.

I have also tried to republish plugin under a different name `com.example.simply007.kontent.delivery.generator` and I haven't found a way to remove it...

I am submitting an issue because I have found this advice here: https://discuss.gradle.org/t/plugin-pending-approval-questions/35017.

### Steps to Reproduce 

Any published plugin is pending approval.

**Comment by JLLeitschuh** [comment]
Hi @Simply007 Can you resubmit your plugin? Also, check your email/spam for an email from us.

**Comment by Simply007** [comment]
Hello @JLLeitschuh,

we needed to move forward with this, so I have found [another way to use the Java code in the Gradle task](https://github.com/Kentico/kontent-java-packages/tree/monorepo-merge/sample-app-spring-boot#generated-strongly-typed-models-with-view-models-transformation-using-thymeleaf-template-engine) so now the best practice would be to define the [task like that](https://github.com/Kentico/kontent-java-packages/blob/monorepo-merge/sample-app-spring-boot/build.gradle#L48) and it eliminates the necessity to have another package in the repository.

I makes also maintenance more convenient since the plugin was the only package of the whole organizational Github written in groovy.

I have double-checked the email and junk and I haven't found any email.
